lib/page-object/accessors.rb

```diff
@@ -637,6 +637,36 @@ def cell(name, identifier={:index => 0}, &block)
     end
     alias_method :td, :cell
 
+
+    #
+    # adds three methods - one to retrieve the text from a table row,
+    # another to return the table row element, and another to check the row's
+    # existence.
+    #
+    # @example
+    #   row(:sums, :id => 'sum_row')
+    #   # will generate 'sums', 'sums_element', and 'sums?' methods
+    #
+    # @param [Symbol] the name used for the generated methods
+    # @param [Hash] identifier how we find a cell.  You can use a multiple parameters
+    #   by combining of any of the following except xpath.  The valid keys are:
+    #   * :class => Watir and Selenium
+    #   * :css => Watir and Selenium
+    #   * :id => Watir and Selenium
+    #   * :index => Watir and Selenium
+    #   * :text => Watir only
+    #   * :xpath => Watir and Selenium
+    #   * :css => Watir and Selenium
+    # @param optional block to be invoked when element method is called
+    #
+    def tr(name, identifier={:index => 0}, &block)
```

Comment:
The corresponding method for td elements is cell. To keep with the past pattern, this should probably be `row`